docs/src/components/home/CompaniesGrid.tsx

```diff
@@ -0,0 +1,69 @@
+import * as React from 'react';
+import Grid from '@material-ui/core/Grid';
+
+export const CORE_COMPANIES = [
```

Comment:
Can we add each company website url so the logo is clickable? We could also make it blue-500 on hover.

Reply:
I'm still doubt on adding link to other page. @oliviertassinari what do you think?

Reply:
What do you think about it? What is getting you in doubt?

Reply:
Adding link could distract user to visit other site, and the companies logo are obvious (most people know what they do). Last, what links? their homepage, or docs, or which products?

Reply:
If we have a link to the product that they use material-ui, then I'm agree.

Reply:
Let's have this as a future idea, then. I think we won't be able to get links for every company displayed about how they're using MUI. At least for now. But I'll keep this idea close to heart so we can do it sometime :)

Reply:
In most cases, we won't be able to link where they use the components, per the nature of our product and of these companies. 👍 for keep the logo inactive, to keep users on the page.